.github/workflows/deploy.yml

```diff
@@ -26,6 +26,7 @@ jobs:
       uses: dawidd6/action-download-artifact@v2
       with:
         workflow: manifold.yml
+        workflow_conclusion: completed
         pr: ${{github.event.pull_request.number}}
```

Comment:
@elalish how does this work? you trigger on push, but here look for `github.event.pull_request`

Came here from https://github.com/dawidd6/action-download-artifact/issues/224, maybe it's related.

Reply:
Huh, good point. I think I had figured every pull request has an associated push, but maybe I'm wrong?